src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,57 @@
+package duke;
+
+import duke.command.Command;
```

Comment:
I really like your abstraction and encapsulation of different commands. Good structure!